docs/developer-docs/latest/setup-deployment-guides/configurations/optional/functions.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Functions - Strapi Developer Documentation
+description: 
+---
+
+<!-- TODO: update SEO -->
+
+# Functions
+
+The `./config/functions/` folder contains a set of JavaScript files in order to add dynamic and logic based configurations.
```

Comment:
We should probably confirm with @alexandrebodin this folder path still exists as I'm honestly not sure